src/app/cli/src/graphql.ml

```diff
@@ -150,6 +150,22 @@ module Types = struct
         ; enum_value "SYNCED" ~value:`Synced
         ; enum_value "OFFLINE" ~value:`Offline ]
 
+  let transaction_status : ('context, Transaction_status.State.t option) typ =
+    enum "TransactionStatus" ~doc:"Status of a transaction"
+      ~values:
+        Transaction_status.State.
+          [ enum_value "INCLUDED" ~value:Included
+              ~doc:"A transaction that is on the longest chain"
+          ; enum_value "PENDING" ~value:Pending
+              ~doc:
+                "A transaction either in the transition frontier or in \
+                 transaction pool but is not on the longest chain"
+          ; enum_value "UNKOWN" ~value:Unknown
+              ~doc:
```

Comment:
Maybe just "The transaction has either been snarked, reached finality through consensus or has been dropped"